backend/backend/core/tools.py

```diff
@@ -1,4 +1,5 @@
 import os
```

Comment:
The import order has changed. This may cause issues with other tools or scripts that rely on the original import order.